plugins/vds3.py

```diff
@@ -189,6 +206,11 @@ def reload(self):
 
         super().reload()
 
+    @asyncthread
+    def download_file(self, row, savepath):
+        f=list(self.open_rows([row]))[0].source
+        f.fs.download(f.given, str(savepath))
```

Comment:
I _think_ we'll be able to use `self.fs.download` here to avoid needing `open_rows`. Should be able to use `row['Key']` for the remote path, and strip any parent dirs off to get a local filename.